doc/local-pv-allocation.md

```diff
@@ -0,0 +1,128 @@
+# Local PV Allocation
+The purpose of this document is to show how to provision a VM disk on a specific node to be able to run it on the same node.
+
+### Prerequesites
+You have a Kubernetes cluster up and running with CDI installed.
+
+### Create your PersistentVolume manifest
+First create a PersistentVolume yaml file with NodeAffinity field to allocate the PV on specific node you choose. Verify 
+that the label you sepcify in the NodeAffinity exists on the node you want the PV to be allocated.
+In addition, add a specific label to be matched by a PersistenVolumeClaim during the binding process.
+An example of such PersistentVolume is:
+
+```yaml
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: local-pv-allocation
+  labels:
+    node: node02
+spec:
+  storageClassName: manual
```

Comment:
storageClassName: manual #This should be the name of your local-volume storage class.